examples/produce_assembly_assy/partcad.yaml

```diff
@@ -20,4 +20,7 @@ assemblies:
 render:
   svg:
     prefix: ./
+    marginTop: 100
```

Comment:
I noticed with cadquery documentation that svg will have many custom options for export. But i am not sure STL, DXF, etc formats will have same options. so only i restricted to SVG. is it will work for other formats too?. if yes then i will apply globally(render_getopts) function itself.